pkg/network/tracer/offsetguess/tracer.go

```diff
@@ -735,14 +740,21 @@ func getUDP6Conn(flowi6 bool) (*net.UDPConn, error) {
 		return nil, nil
 	}
 
-	linkLocal, err := GetIPv6LinkLocalAddress()
+	linkLocals, err := GetIPv6LinkLocalAddress()
 	if err != nil {
 		// TODO: Find a offset guessing method that doesn't need an available IPv6 interface
 		log.Debugf("unable to find ipv6 device for udp6 flow offset guessing. unconnected udp6 flows won't be traced: %s", err)
 		return nil, nil
 	}
-
-	return net.ListenUDP("udp6", linkLocal)
+	var conn *net.UDPConn
+	var err error
+	for _, a := range linkLocals {
```

Comment:
```suggestion
	for _, linkLocal := range linkLocals {
```
maybe clearer name? or `addr`, `linkLocalAddr`?